### PR TITLE
[8.x] Fix docblocks, imports 

### PR DESCRIPTION
**Pull request by iamgergo**

### CODE REVIEW

**Review by driesvints — APPROVED** [review]

File: src/Illuminate/Testing/ParallelRunner.php [hunk 1]
```diff
@@ -126,6 +126,8 @@ protected function forEachProcess($callback)
      * Creates the application.
      *
      * @return \Illuminate\Contracts\Foundation\Application
+     *
+     * @throws \RuntimeException
```
Comment (GrahamCampbell) [inline src/Illuminate/Testing/ParallelRunner.php:130]: RuntimeException and Error don't usually show in the throws doc. One normally assumes they can always be thrown by any code.

File: src/Illuminate/Queue/Console/RetryCommand.php [hunk 2]
```diff
@@ -150,6 +150,8 @@ protected function resetAttempts($payload)
      *
      * @param  string  $payload
      * @return string
+     *
+     * @throws \RuntimeException
```
Comment (GrahamCampbell) [inline src/Illuminate/Queue/Console/RetryCommand.php:154]: same

File: src/Illuminate/Queue/CallQueuedHandler.php [hunk 3]
```diff
@@ -88,6 +88,8 @@ public function call(Job $job, array $data)
      *
      * @param  array  $data
      * @return mixed
+     *
+     * @throws \RuntimeException
```
Comment (GrahamCampbell) [inline src/Illuminate/Queue/CallQueuedHandler.php:92]: same

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: src/Illuminate/Testing/ParallelRunner.php (hunk 1 above)
Comment (iamgergo) [reply src/Illuminate/Testing/ParallelRunner.php:130]: I see. There are already 60+. So should I delete them (that I've added) or leave them as they are?

Comment (GrahamCampbell) [reply src/Illuminate/Testing/ParallelRunner.php:130]: Just don't add any more unnecessary ones.

**Pull request merged**